### DISCUSSION TITLE
Can't create reaction "This field is required" to filled field

### DISCUSSION THREAD

**Discussion by checklister** [SecurityBrewery/catalyst#1093] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/19972825-ca01-440c-b722-4d1d226a4ffb)
I can't hit the "Save" button. If I enter some string value (except following on the screen) in these fields, it becomes permanent empty.

**Comment by cugu** [comment, marked as answer]
Thanks for the report @checklister ! 

I just released a fix in [v0.13.2](https://github.com/SecurityBrewery/catalyst/releases/tag/v0.13.2) which should solve this.
